### DISCUSSION TITLE
[语法高亮]Chrome浏览器下Golang语法高亮失败

### DISCUSSION THREAD

**Discussion by shiqinfeng1** [tangly1024/NotionNext#1988] [Q&A]
**描述bug**
相同的golang语言代码段在edge下可以被高亮，但是在chrome下不能被高亮

**复现步骤**
出现这个bug的操作步骤

**期望的正常结果**
希望按这个步骤，正常操作结果是什么

**截图**
chrome下：
![image](https://github.com/tangly1024/NotionNext/assets/7941301/be8dc72c-9770-4167-b9d4-93f2b5c4f39d)
edge下：
![image](https://github.com/tangly1024/NotionNext/assets/7941301/102c2485-514c-4a6f-a727-5112e338770c)

**环境**

- 操作系统: [ windows]
- 浏览器 [例如. chrome,]
- NotionNext版本 [最新版4.0.15]
- 主题 [例如. next]

**补充说明**
出现问题的chrome版本是：windows版 109.0.5414.75（正式版本） （32 位）
在mac版的chrome上试了一下可以高亮，版本：mac版 117.0.5938.132（正式版本） (x86_64)
<img width="834" alt="image" src="https://github.com/tangly1024/NotionNext/assets/7941301/b14e24d0-fd20-47c2-8ec5-ffc51cd948eb">

**Comment by tangly1024** [comment]
感谢反馈
还需要提供一下出现问题的文章页面

**Comment by shiqinfeng1** [comment]
> 感谢反馈 还需要提供一下出现问题的文章页面

可以看下这个页面 https://www.slw.ac.cn/article/mysql.html     

这是edge下 ： ![image](https://github.com/tangly1024/NotionNext/assets/7941301/9b8ab942-4663-4be2-88b7-e19e0e5228a4)

这是chrome下： ![image](https://github.com/tangly1024/NotionNext/assets/7941301/ecc94dc1-3b33-4622-aa32-425c396e4021)

**Comment by tangly1024** [comment]
嗯， 它在我的Chrome上工作正常 版本 117.0.5938.152（正式版本） （64 位）

![图片](https://github.com/tangly1024/NotionNext/assets/15920488/bbe700ed-c115-4f6c-bc9c-19d34cdf3877)

问题出在哪呢，[思考中...]

**Comment by shiqinfeng1** [comment]
我升级chrome吧

> 嗯， 它在我的Chrome上工作正常 版本 117.0.5938.152（正式版本） （64 位）
> 
> ![图片](https://user-images.githubusercontent.com/15920488/276195920-bbe700ed-c115-4f6c-bc9c-19d34cdf3877.png)
> 
> 问题出在哪呢，[思考中...]

我升级chrome吧
